### PR TITLE
[image_picker_for_web] Can't select some video file types (3gp, mp4, possibly others...)

### PR DESCRIPTION
**Pull request by Makapre**
## Description

This PR attempts [#58532](https://github.com/flutter/flutter/issues/58532) and solves video selecting issues when using the image_picker plugin in safari. This issue comes from the webkit and apples own MIMETypes. ([see](https://trac.webkit.org/browser/trunk/Source/WebCore/platform/MIMETypeRegistry.cpp))
Following video file types can be selected(others not tested):

- .mp4
- .3gp
- .avi
- .mov

## Related Issues

* fixes https://github.com/flutter/flutter/issues/58532

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart [hunk 1]
```diff
@@ -8,7 +8,7 @@ import 'package:image_picker_platform_interface/image_picker_platform_interface.
 final String _kImagePickerInputsDomId = '__image_picker_web-file-input';
 final String _kAcceptImageMimeType = 'image/*';
 // TODO The value below seems to not be enough for Safari (https://github.com/flutter/flutter/issues/58532)
```
Comment (ditman) [inline packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart:10]: Please remove this comment, it seems this change makes it obsolete!

**Review by ditman — CHANGES_REQUESTED** [review]
Please, modify the pubspec.yaml and the CHANGELOG.md so we can publish this change to pub.dev.

File: packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart (hunk 1 above)
Comment (Makapre) [reply packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart]: I removed the comment 👍🏻

**Comment by Makapre** [comment]
Latest commit updates pubspec.yml and CHANGELOG.md.

**Review by ditman — APPROVED** [review]
Looks good to me! Thanks for the contribution. Let's wait for the CI to finish in order to merge this.

**Comment by ditman** [comment]
Merged and published! Thanks @Makapre!

**Pull request merged**